addons/web_editor/static/src/js/editor/odoo-editor/test/spec/list.test.js

```diff
@@ -60,6 +60,15 @@ describe('List', () => {
                                 '<ul><li><span><b>ab</b></span> <span><i>cd</i></span> ef[]gh</li></ul>',
                         });
                     });
+                    it('should not turn a div into a list', async () => {
+                        await testEditor(BasicEditor, {
+                            contentBefore:
+                                '<div>ab[]cd</div>',
+                            stepFunction: toggleUnorderedList,
+                            contentAfter:
+                                '<div>ab[]cd<ul><li></li></ul></div>',
+                        });
```

Comment:
Considering the result of `<p>ab[]cd</p>` + `toggleList`, I would have expected a different result:
```html
<div>
    <ul>
        <li>ab[]cd</li>
    </ul>
</div>
```
What do you think @SimonGenin ?

Reply:
That means to put the content of the div inside of the li. But if the content of the div is an other div, or something complex (not just some text), then problem is still there, no ?